src/components/Card.tsx

```diff
@@ -47,8 +68,18 @@ const Card = ({ cardData }: CardProps) => {
           <div>
             {isCollected({ cards: collection.cards, id: cardData.id }) ? (
               <div>
-                {pathname === '/search' ? <p>{t('In Collection')}</p> : null}
+                {pathname === '/search' ? (
+                  <InCollection
+                    transition={{ transition: 0.5 }}
```

Comment:
The add button also flashes it's finished state before playing the animation sometimes